client/post-editor/editor-action-bar/style.scss

```diff
@@ -9,11 +9,33 @@
 	@include breakpoint( "<660px" ) {
 		margin-bottom: 8px;
 	}
+}
+
+.editor-status-label.editor-action-bar__saved-status {
```

Comment:
Can't you target this with `.editor-action-bar .editor-status-label` instead of creating a new class?